packages/contracts-bedrock/contracts/L1/L1CrossDomainMessenger.sol

```diff
@@ -13,22 +13,33 @@ import { CrossDomainMessenger } from "../universal/CrossDomainMessenger.sol";
  *         interface instead of interacting with lower-level contracts directly.
  */
 contract L1CrossDomainMessenger is CrossDomainMessenger {
+    /**
+     * @notice Contract version number.
+     */
+    uint8 public constant VERSION = 1;
```

Comment:
Any reason for not also making this change to the L2 contracts which AFAIUI will also be placed behind an ERC1967 proxy?

Reply:
I'll do the same to the L2 contracts, just wanted to keep this PR relatively small and make sure there was agreement on the direction

Reply:
Agreed